libraries/octree/src/OctreeUtils.h

```diff
@@ -27,4 +27,10 @@ float boundaryDistanceForRenderLevel(unsigned int renderLevel, float voxelSizeSc
 
 float getAccuracyAngle(float octreeSizeScale, int boundaryLevelAdjust);
 
+const float MIN_ELEMENT_APPARENT_ANGLE = 0.0087266f; // ~0.57 degrees in radians
+// NOTE: the entity bounding cube is larger than the smallest containing octree element by sqrt(3)
+const float SQRT_THREE = 1.73205080f;
+const float MIN_ENTITY_APPARENT_ANGLE = MIN_ELEMENT_APPARENT_ANGLE * SQRT_THREE;
+const float MIN_VISIBLE_DISTANCE = 0.0001f; // helps avoid divide-by-zero check
+
```

Comment:
The `MIN_ELEMENT_APPARENT_ANGLE` is the fundamental dial for when to cull elements and entities for LOD.  This may need some tuning but not much.  The old code had a cutoff of 400m for a 1x1x1 octree element, which corresponds to: `apparentAngle = sqrt(3)/400 = 0.0043301270`, which is half the size of this setting.

Note: an interface-client could decrease its LOD by setting the `lodLevelOffset` to something larger than 0.  Each increment corresponds to a doubling of the effective `MIN_ELEMENT_APPARENT_ANGLE`.